lib/scala/tree/src/main/scala/edu/rpi/tw/twxplore/lib/geo/models/domain/Borough.scala

```diff
@@ -0,0 +1,12 @@
+package edu.rpi.tw.twxplore.lib.geo.models.domain
+
+import edu.rpi.tw.twks.uri.Uri
+import edu.rpi.tw.twxplore.lib.base.utils.rdf.{Rdf, RdfReader, RdfWriter}
+import org.apache.jena.geosparql.implementation.vocabulary.Geo
+import org.apache.jena.rdf.model.{Resource, ResourceFactory}
+import org.apache.jena.vocabulary.{RDF, RDFS}
+
+import scala.collection.mutable.ListBuffer
+
+final case class Borough(borough: String, borocode: Int, ntaList: ListBuffer[NTA])
```

Comment:
Using mutable containers (ListBuffer) is bad enough. Passing them around is really bad. At the very least, convert to a mutable List.

You also don't need to suffix variables in a typed language.

final case class Borough(name: String, code: Int, ntas: List[NTA])

Reply:
Im not too sure if I fixed it? I changed the types to Lists instead of ListBuffers.. but I also changed the declaration of the List to be vars instead of val.. not too sure how to get around the issue, as I still need to append to the list as I find new instances going through the CSV (ie.. finding "Queens Borough" while going through the CSV  "appends" to a temporary ListBuffer variable at the moment, which in turn is then converted to a List and reassigned to the boroughs list of City)

Reply:
Have to keep separate mutable state mapping borough -> ListBuffer. Passing immutable state to a case class is a no-no.